backend/useragent/user_agent.go

```diff
@@ -45,6 +45,11 @@ func Parse(s string) (*UserAgent, error) {
 	}, nil
 }
 
+// Empty creates a new UserAgent with default values.
+func Empty() (*UserAgent, error) {
+	return New("0.0.0", "unknown", "unknown")
```

Comment:
I think we could probably skip using `New` here (to avoid ignoring the error) and just create the struct ourselves

Reply:
That's much better, fixed in 8bd76d8 👍